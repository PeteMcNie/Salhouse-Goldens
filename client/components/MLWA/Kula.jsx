import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'
import Image from 'react-bootstrap/Image'

function Kula () {
  return (
    <>
      <Container>
        <Row className='row'>
          <Col>
            <h2 className='bodyTitle'><a href='/kula' className='doglink'>Kula</a></h2>
          </Col>
        </Row>
        <Row className='row'>
          <Col lg={6} className='leftCol'>
            <p>
              Sandy wasn’t having any puppies at that time but said she would take my number and contact me in the future. Well,
              time went by and in fact I passed Sandy one day while I was walking Phoenix down the river. I only recognised her
              by the fact she had three Goldens as I had never met her up close. We had a little chat and she remembered talking
              to me on the phone. One of her dogs was pregnant but was having a few problems. So I kept that in my mind and I
              can’t remember whether I phoned Sandy or whether she phoned me later on, but anyway, we were in contact about a
              week after her bitch had had the puppies. She still had a puppy available to a good home as there were ten puppies
              in the litter and she had thought that I was a suitable owner for one. I was so excited about this. Another puppy
              to play with Phoenix. Yeah.
            </p>
            <p>
              Sandy was so good with her owners. We were allowed to go round at any time to see them. She would chat and chat
              about dogs and that suited me just fine as I was never happier than when talking about dogs. My boys and Rob came
              too and became just as excited about the new puppy. The boys loved to sit in with the puppies and have them jump
              all over them and try to eat their toes and loose clothing etc.
            </p>
            <p>
              There were four bitches in the litter and I definitely wanted a female. As I was the last person to be a home for
              a puppy, I was to get last choice of puppy from the litter and Sandy didn’t believe in letting puppies be selected
              by owners until their were about 5 weeks of age, when a bit of personality was showing through. So we played with
              all the female puppies and wondered which one was going to be ours. Eventually the time came where the three other
              female puppies had been chosen, so we now had a puppy that we knew we were going to take home one day. We needed a
              name and by this time computers were around and it was quite easy to look up names on that. I decided that I would
              like a name that had something to do with the word “golden”. We eventually found a lovely name, Kula, which means
              “the golden one” in Hawaiian. That seemed perfect, Kula it was.
            </p>
          </Col>
          <Col lg={6}>
            <div>
              <Image src="images/fourFemalePuppies.jpg" fluid className='maddie' />
              <p className='MLWAimgtext'>The four female puppies in the litter - one is Kula!</p>
            </div>
          </Col>
        </Row>
        <Row className='row'>
          <Col lg={6} className='leftCol'>
            <p>
              So Kula joined our family and there was a definite change in Phoenix. Before we got Kula, Phoenix had been an only
              dog and although I took her for long walks every day, she would only run occasionally if we ran. Mostly she would be
              happy trotting along with us. At home, she would lounge around a lot of the time and play with us and the boys
              occasionally when we had some fun and games. However, once Kula entered our house, Phoenix’s life of leisure changed.
              She was fascinated with this puppy and wanted to play with her from the moment Kula arrived home. I have never known
              a dog who is so gentle as Phoenix. She tried her hardest to play with Kula gently, only occasionally accidentally
              treading on her. She allowed Kula to bite her legs and ears and they spent hours together playing. This new puppy
              meant that Phoenix had competition too, so she had to liven up a bit. Overnight she became much more animated. She
              was always on the lookout for where Kula was in the house and whether she was getting some attention from the humans
              or not. If Kula was getting attention, Phoenix would get up and bring a tug toy to us and barge in as if to say,
              “Hey, you should be playing with me”.
            </p>
          </Col>
          <Col lg={6}>
            <div>
              <Image src="images/kula.jpg" fluid className='maddie' />
            </div>
          </Col>
        </Row>
        <Row className='row'>
          <Col lg={6} className='leftCol'>
            <p>
              I started to do agility with Kula initially and then Phoenix. Phoenix turned out to be much more willing and keen.
              We did this for a couple of years and had some fun.
            </p>
            <p>
              Once I had Kula, Sandy and I started to go for walks together as we lived fairly close.  We spent hours talking about
              dogs and other things. One thing that I mentioned to Sandy one day was that I would love to have puppies myself. I
              remember when I was young that one or two people at school had dogs that had puppies at home, and I was always very
              jealous and wondered what it would be like. Sandy guided me through getting Kula’s hips and elbows scored and eyes
              tested and when the time was right, we decided to have a litter.
            </p>
          </Col>
          <Col lg={6}>
            <div>
              <Image src="images/lastJumpAgility.jpg" fluid className='maddie'/>
            </div>
          </Col>
        </Row>
        <Row className='row'>
          <Col>
            <p>
              We had a litter of seven puppies in February 2003, had great fun caring for them until they were old enough to go to
              homes and kept one, <a href='/shadow'>Shadow</a>. Shadow has been the most beautiful family dog ever. She is a singer - singing for joy
              when she greets us in the morning or after we&apos;ve been out and then singing when she meets people she knows. We have
              to try and tone her down sometimes as she is so vocal.  I got into breed showing with Shadow - many thanks to Jan
              Cook who helped me with guidance and support with this.
            </p>
            <p>
              In April 2010 Shadow had a litter of puppies out of which we kept another beautiful female who we called <a href='/sherry'>Sherry</a>.
              Sherry is an absolutely wonderful personality - she is busy and interactive and a fun dog. She is such a communicator
              and loves to stalk the family and the other dogs.
            </p>
            <p>
              We are so lucky to have these four lovely dogs at this stage in our life. I can&apos;t imagine being without them although
              I am aware that Phoenix and Kula are getting on a bit now. Phoenix in particular struggles with being nearly blind
              now and also nearly deaf. She still loves her daily walks, although I am often found running off in another direction
              to go and get her when she has &apos;lost&apos; us. She doesn&apos;t swim any more but loves to bathe in the shallows and roll in
              the grass.
            </p>
            <p>
              We have these beautiful creatures for far too short a time. I am currently reading about the psychology of dogs and
              I believe we owe it to them to try and understand them better. I am interested in the science, what they have proved
              is the right way to live with and train a dog, not the myths of domination and punishment that has been promoted
              previously. I think dog training will alter dramatically in the next few years to the benefit of both human and
              canines. If you are interested in reading a book on the science of how dogs learn etc, have a look at
              <a href='https://www.youtube.com/watch?v=-R14Hsd1RtI'> this video</a> by John Bradshaw and I would recommend you pick
              up a copy of the book In Defence of Dogs.
            </p>
          </Col>
        </Row>
        <hr className='sectionSeparator' />
      </Container>
    </>
  )
}

export default Kula
