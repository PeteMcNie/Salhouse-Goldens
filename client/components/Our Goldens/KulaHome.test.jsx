import React from 'react'
import { render } from '@testing-library/react'

import Kula from './KulaHome'

test('Kula component matches snapshot', () => {
  const { container } = render(<Kula />)
  expect(container).toMatchSnapshot()
})
