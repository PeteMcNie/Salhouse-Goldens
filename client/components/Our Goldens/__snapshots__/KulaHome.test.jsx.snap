// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kula component matches snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo.jpeg"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <nav
    class="belowNavbar nav nav-tabs"
    role="tablist"
  >
    <a
      aria-controls="controlled-tab-example-tabpane-kulaHome"
      aria-selected="true"
      class="nav-item nav-link active"
      data-rb-event-key="kulaHome"
      href="#"
      id="controlled-tab-example-tab-kulaHome"
      role="tab"
    >
      Kula
    </a>
    <a
      aria-controls="controlled-tab-example-tabpane-KulaLitter"
      aria-selected="false"
      class="nav-item nav-link"
      data-rb-event-key="KulaLitter"
      href="#"
      id="controlled-tab-example-tab-KulaLitter"
      role="tab"
      tabindex="-1"
    >
      Kula's Litter
    </a>
  </nav>
  <div
    class="tab-content"
  >
    <div
      aria-hidden="false"
      aria-labelledby="controlled-tab-example-tab-kulaHome"
      class="fade tab-pane active show"
      id="controlled-tab-example-tabpane-kulaHome"
      role="tabpanel"
    >
      <div>
        <h2>
          Kula
        </h2>
      </div>
      <div>
        <h3>
          Sanastasia Sth'n Star Chara (Kula)
        </h3>
        <p>
          Sire: CH Seacove Southern Cross
        </p>
        <p>
          Dam: Sanastasia Treasured Teah
        </p>
        <p>
          5 June 2003 to 21 October 2019
        </p>
        <p>
          Hips: 7:8  Elbows: B:B  Eyes: Clear Eye Certificate 20/5/12
        </p>
        <div>
          <img
            class="img-fluid"
            src="images/kula1.jpg"
          />
        </div>
      </div>
      <div>
        <h4>
          Achievements:
        </h4>
        <p>
          Kula's greatest achievement was to have a lovely litter of puppies in 2006. She had a difficult pregnancy - for me anyway, as almost from the moment she got pregnant she wouldn't eat anything that had the word “petfood“ on it. I can only guess that she had morning sickness throughout this pregnancy and it was a real challenge to find things for her to eat. If I found that she would eat fresh fish one day, I would buy in lots of it only to find that after about three days, she no longer wanted to eat that. So we would go on to something else - steak, sausages cooked on the barbecue, chicken, leftover food from our meals, eggs, cheese etc - only to find that in each occasion, after one or two days she would no longer eat that food. And its not as if we could go back to the same food a week or two later. It seemed that once she had eat a type of food it was then marked as “never to touch again while I am pregnant“. She drove me almost to despair when, at the end of her pregnancy she stopped eating totally for three days. I couldn't get her tempted at all and we watched the puppies seeming to get larger and Kula to get smaller. When she finally had the litter they were very healthy puppies and Kula, who was absolutely skin and bone by then, started to eat immediately, initially with half a pot of honey while she was in labour and then eating everything possible from then onwards. What a relief!
        </p>
        <p>
          Kula is my only dog to have clicked with the retrieve “thing“. She caught on at an early age that if she brought back the stick or the ball to my feet, she would have the fun of another chase - an achievement that sadly kula and Shadow seem to have missed out on. Kula is now going to try to do a bit of obedience. I think she has potential but we probably should have started a few years ago. However, she is pretty clever at picking things up and we can have some fun along the way.
        </p>
        <p>
          To find out more about how Kula came into our lives, go to the 
          <a
            href="/mylifewithanimals"
          >
            My Life with Animals
          </a>
           page.
        </p>
        <p>
          Kula's Litter 2006.
        </p>
      </div>
      <div>
        <div
          class="table-responsive-sm"
        >
          <table
            class="table table-sm table-bordered"
          >
            <thead>
              <tr>
                <th
                  colspan="3"
                >
                  Pedigree of Sanastasia Sth'n Star Chara
                </th>
              </tr>
            </thead>
            <tbody>
              <tr>
                <td>
                  Parents
                </td>
                <td>
                  Grandparents
                </td>
                <td>
                  Great Grandparents
                </td>
              </tr>
              <tr>
                <td
                  rowspan="4"
                >
                  Sire: CH Seacove Southern Cross
                </td>
                <td
                  rowspan="2"
                >
                  CH Glentreve Frozen-Asset
                </td>
                <td>
                  Eng. CH Paudell Easter Plantagenet at Kerrien (UK)
                </td>
              </tr>
              <tr>
                <td>
                  CH McIlroy Interlude
                </td>
              </tr>
              <tr>
                <td
                  rowspan="2"
                >
                  Seacove Bireme
                </td>
                <td>
                  GR & Aust. CH Caldicot Swift
                </td>
              </tr>
              <tr>
                <td>
                  Goldriver Amber-Sunrise QC CDX
                </td>
              </tr>
              <tr>
                <td
                  rowspan="4"
                >
                  Dam: Sanastasia Treasured Teah CDX
                </td>
                <td
                  rowspan="2"
                >
                  OB GR CH Speyside Fidgety Feet CDX
                </td>
                <td>
                  GR CH Graveleigh Soaring High QC CDX
                </td>
              </tr>
              <tr>
                <td>
                  Speyside Shades of Gold CDX
                </td>
              </tr>
              <tr>
                <td
                  rowspan="2"
                >
                  Sanastasia Special Lady CDX
                </td>
                <td>
                  Sanastasia Gold Falkor
                </td>
              </tr>
              <tr>
                <td>
                  Speyside Silver Lining CDX
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
      <hr />
      <div>
        <p>
          Photos:
        </p>
        <img
          class="img-fluid"
          src="images/kula2.jpg"
        />
        <p>
          Kula likes to snooze in the bean bag!
        </p>
        <img
          class="img-fluid"
          src="images/kula3.jpg"
        />
        <p>
          Kula posing.
        </p>
        <img
          class="img-fluid"
          src="images/kula4.jpg"
        />
        <p>
          Posing again, but upside down this time!
        </p>
        <img
          class="img-fluid"
          src="images/kula5.jpg"
        />
        <p>
          Kula after a swim at Lake Taupo.
        </p>
        <img
          class="img-fluid"
          src="images/kula6.jpg"
        />
        <p>
          Such a hot day, the only thing for it was to swim up the river.
        </p>
        <img
          class="img-fluid"
          src="images/kula7.jpg"
        />
        <p>
          Kula (right) and Shadow enjoy time in the sun.
        </p>
        <img
          class="img-fluid"
          src="images/kula8.jpg"
        />
        <p>
          Kula ready for a walk!
        </p>
        <img
          class="img-fluid"
          src="images/kula9.jpg"
        />
        <p>
          Kula's Mum - Teah.
        </p>
      </div>
    </div>
    <div
      aria-hidden="true"
      aria-labelledby="controlled-tab-example-tab-KulaLitter"
      class="fade tab-pane"
      id="controlled-tab-example-tabpane-KulaLitter"
      role="tabpanel"
    >
      <div>
        <h2>
          Kula's Litter 2006
        </h2>
      </div>
      <div>
        <img
          class="img-fluid"
          src="images/kl1.jpg"
        />
        <p>
          On 21 February 2006, Kula (pictured left top) had a litter of 7 beautiful puppies, 5 boys and 2 girls.
        </p>
        <img
          class="img-fluid"
          src="images/kl2.jpg"
        />
        <p>
          The father of these puppies is Jordan 
          <a
            href="https://www.mistymoorgoldens.com/"
          >
            (NZ CH Santamaria The Patriot)
          </a>
           (pictured left bottom).
        </p>
      </div>
      <hr />
      <div>
        <p>
          Photos:
        </p>
        <img
          class="img-fluid"
          src="images/kl3.jpg"
        />
        <p>
          Kula 8 weeks pregnant (like a beached whale!).
        </p>
        <img
          class="img-fluid"
          src="images/kl4.jpg"
        />
        <p>
          Our first ever puppy born!
        </p>
        <img
          class="img-fluid"
          src="images/kl5.jpg"
        />
        <p>
          We had a lot of fun with these puppies and of course kept the lovely Shadow.
        </p>
        <p>
          The puppies all had wonderful homes to go to and are still currently enjoying their life to the full.
        </p>
        <img
          class="img-fluid"
          src="images/kl6.jpg"
        />
        <p>
          Kula wonders what is moving under the blanket!
        </p>
        <img
          class="img-fluid"
          src="images/kl7.jpg"
        />
        <p>
          Pileup of puppies!
        </p>
        <img
          class="img-fluid"
          src="images/kl8.jpg"
        />
        <p>
          Puppies feeding.
        </p>
        <img
          class="img-fluid"
          src="images/kl9.jpg"
        />
        <p>
          Asleep under the barbecue.
        </p>
        <img
          class="img-fluid"
          src="images/kl10.jpg"
        />
        <p>
          All on Jeff!
        </p>
        <img
          class="img-fluid"
          src="images/kl11.jpg"
        />
        <p>
          Let's try out the pool.
        </p>
        <img
          class="img-fluid"
          src="images/kl12.jpg"
        />
        <p>
          1st birthday - all except the second female here.
        </p>
      </div>
    </div>
  </div>
</div>
`;
